### ISSUE TITLE
Create a thank you page

### ISSUE THREAD

**Issue by vaibhav-kaushal** [techrail/devta#101]
We must give credit to the third part libraries we have used!

**Comment by pratyushtiwary** [comment]
Suggestion: Automatic generation of thank you page using package.json

If it's ok then I can do this

**Comment by vaibhav-kaushal** [comment]
I think we can use only the direct dependencies for it - wouldn't that be good?

**Comment by pratyushtiwary** [comment]
Yup, I'll try to create a script which will list out only the packages that are used inside a module
